### DISCUSSION TITLE
카테고리 분배

### DISCUSSION THREAD

**Discussion by subin1224** [subin1224/Simple-brain-training#26] [Ideas]
# Main 화면

Main 화면에 들어간 후 페이지를 바로 문제를 진행하는 것이 아니라 카테고리를 나누는 것은 어떨까요?

(예시 이미지)
![image](https://github.com/subin1224/Simple-brain-training/assets/76253952/2c240c3f-ff64-4505-a1af-dd088fde2063)
사진: https://humanbenchmark.com/dashboard

**Comment by mmeat512** [comment]
카테고리로 나눌만큼 문제가 많지 않다고 생각합니다.
사용자입장에서 불필요한 페이지 이동이 생기게 되지 않을까요?
